### PR TITLE
3rd changes

### PR DESCRIPTION
**Pull request by rtlbddev**

### CODE REVIEW

File: testFile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 1st Changes in this file
 2nd changes just made
 Fixed
+3rd changes from another repo
```
Comment (imamulnayeem) [inline testFile:4]: Please also add 4th changes along with this

File: testFile [hunk 2]
```diff
@@ -1,3 +1,4 @@
 1st Changes in this file
 2nd changes just made
 Fixed
```
Comment (imamulnayeem) [inline testFile:3]: please also remove "Fixed"

File: testFile (hunk 1 above)
Comment (rtlbddev) [reply testFile:4]: ok

**Review by imamulnayeem — CHANGES_REQUESTED** [review]
please add 6th changes

**Pull request merged**